requirements.txt

```diff
@@ -24,3 +24,4 @@ six==1.9.0
 sqlparse==0.1.14
 tablib==0.10.0
 whitenoise==1.0.6
+selenium==2.45.0
```

Comment:
@kave I am going to start writing those alongside the Python tests. Did you want to do them?